insertion_sort/insertion_sort.py

```diff
@@ -0,0 +1,22 @@
+def insertionSort(alist):
+   """
+    Performs a insertion Sort
+    :param alist: Iterable of elements   
```

Comment:
The "Performs a inser.." text has extra whitespace before it..
It is not inline with the `"""` . 
Please fix that.